tasks/framework-tools/frameworkDepsToProject.mjs

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env node
 /* eslint-env node */
 
-import path from 'path'
+import path from 'node:path'
```

Comment:
Why are you doing this here? I'm confused... why is this necessary

Reply:
This is the node protocol. It's just a new feature in ES Modules: https://nodejs.org/api/esm.html#node-imports. There's no meaningful difference; the only justification I have is 
1) using URLs is more web-y than the previous way of doing things 
2) it's easier for beginners (they can tell that this is from the node std lib)

The Node.js docs will most likely use it in the future too: https://github.com/nodejs/node/issues/38343.

Can remove it though; just added it cause I liked it.

Reply:
I like it. Keep us moving in that direction, k? My only concern would be using features that aren't supported in v16 (i.e. > 16 only).